### ISSUE TITLE
Don't show empty lines if DMDImport Timing is enabled but doesn't work

### ISSUE THREAD

**Issue by Cogitri** [Pure-D/serve-d#91]
It appears that if DMDImport Timing is enabled but doesn't work (e.g. if one doesn't use dmd, I suppose) Code-D inserts lines to display the timing information but since that can't be generated these lines stay empty:

![image](https://user-images.githubusercontent.com/8766773/75612438-ac3ec800-5b23-11ea-9c37-031db421b9de.png)

It'd be nicer if Code-D generated an error message and didn't generate these empty newlines which look kind of odd.

**Comment by WebFreak001** [comment]
as per LSP API it is only possible to provide locations for code lenses (the import timings) and then only later on fill them.

It's like this because otherwise the code would jump around if it would resolve them randomly and you would probably misclick somewhere if you clicked while it was resolving or it would be distracting while coding.

I think the blank lines while it is loading is not such an issue. Rather it should be investigated why it doesn't work for you there.

It's disabled by default because measuring the import timing is a blocking task so no other tasks (like auto completion or linting) are running during it, which could affect user experience. I don't think it's the most useful feature now because the import times can differ significantly between runs, making it not very useful to even check timings between each other in relation to each other.

**Comment by Cogitri** [comment]
> Rather it should be investigated why it doesn't work for you there.

I suppose it works only if dmd is available? I only have ldc2 available. Anyway, I can just disable the feature for now.

**Comment by WebFreak001** [comment]
yep that's the issue, that's why it's called "dmdImportTiming" :p

**Comment by Cogitri** [comment]
Yup, expected that, I guess that setting could check for dmd then though?
